src/packages/shared-utils/decode.ts

```diff
@@ -0,0 +1,7 @@
+export const decodeBase64WithUtf8 = (value: string): string => {
+  // Assuming the record value is in base64 encoding
+  const base64String = value;
```

Comment:
Is there a purpose with this variable init? Maybe we can rename `value` to `potentialBase64String`? And on that note, given what I saw in `opensearch-lib`, perhaps we should handle errors in case `value` isn't actually a string!